cmd/local/prerun.go

```diff
@@ -34,8 +34,6 @@ func validateLocal(cmd *cobra.Command, args []string) error {
 
 	config := configs.ReadConfig()
 
-	log.Info().Msg("sending init started metric")
```

Comment:
good call! we have two logs for the same thing. 

when debugging telemetry, it's nice to read the logs to check if the telemetry send function was called, wdyt to update the log message from
```
pkg.InformUser("Sending installation telemetry", silentMode)
```
to
```
pkg.InformUser("Sending kubefirst.init.started installation telemetry", silentMode)
```

p.s: we can update it on a separated issue/PR to also update the other references, there is also constants we can use for telemetry names.

Reply:
we will have telemetry update this week, this can wait.